package solv.fact.service.auth;

public class AuthException extends RuntimeException {

    public AuthException(String message) {
        super(message);
    }

}
